client/src/components/DetailedPatentView.vue

```diff
@@ -130,6 +136,19 @@ export default defineComponent({
             this.patentAvailable = false;
             this.$emit('removeFromSaved');
         },
+        highlightTitle(title: string, keywords: string[]) {
+            const pattern = new RegExp(`(${keywords.join('|')})`, 'gi');
+            return title.replace(pattern, (match) => {
+                return '<mark style="background-color:rgba(245, 255, 129, 1)">' + match + '</mark>';
+            });
+        },
+        highlightAbstract(abstract: string, keywords: string[]) {
+            const pattern = new RegExp(`(${keywords.join('|')})`, 'gi');
+            console.log(pattern);
```

Comment:
This console.log could be removed

Reply:
ah sure that was for testing i forgot to remove